src/Layout/index.jsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
-import { gtaModel } from '../constants/gtaModel';
-import { GtaVersions } from '../Components';
+import { gtaData } from '../data/data';
```

Comment:
you can rename the file in the folder data to index.js and just import data. Also rename the gtaData to just data.